pipeline/4/Jenkinsfile-tier-1-wrapper.groovy

```diff
@@ -61,8 +62,8 @@ node(nodeName) {
 		println "Triggering tier-1 jobs for compose ${composeInfo}"
 	}
 
-	timeout(unit: "HOURS", time: 24) {
-        for(jobName in tier1Jobs) {
+	for(jobName in tier1Jobs) {
+        timeout(unit: "HOURS", time: 5) {
```

Comment:
IMO, let's limit the runtime execution to 24 hours and not cap the individual jobs... What do you think?

Reply:
Sure